src/test/scala/edu/ie3/simona/model/thermal/ThermalGridWithStorageOnlySpec.scala

```diff
@@ -130,18 +132,16 @@ class ThermalGridWithStorageOnlySpec
         )
         val state = HpState(
           10800, // after three hours
-          testGridAmbientTemperature,
           gridState,
-          HpOperatingPoint(zeroKW, ThermalOpWrapper.zero),
-          testGridAmbientTemperature,
+          HpOperatingPoint(zeroKW, ThermalGridOperatingPoint.zero),
           onlyAdditionalDemandOfHeatStorage,
         )
 
         val updatedThermalGridState =
-          thermalGrid.updatedThermalGridState(
+          thermalGrid.determineThermalGridState(
             state.tick,
             state,
```

Comment:
This does nothing, right? Last state is given at the current tick

Reply:
Right 😄